opencti-platform/opencti-front/src/private/components/import/ImportContent.jsx

```diff
@@ -411,12 +418,12 @@ class ImportContentComponent extends Component {
                       </Tooltip>
                       <ListItemText
                         primary={connector.name}
-                        secondary={R.join(',', connector.connector_scope)}
+                        secondary={truncate(connector.connector_scope.join(','), 30)}
```

Comment:
it might be useful to have a tooltip around this truncated content, to see the full content on mouse hover.